seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/config/FakeConfig.java

```diff
@@ -27,11 +27,11 @@
 @Builder
 @Getter
 public class FakeConfig implements Serializable {
-    private static final String ROW_NUM = "row.num";
-    private static final String MAP_SIZE = "map.size";
-    private static final String ARRAY_SIZE = "array.size";
-    private static final String BYTES_LENGTH = "bytes.length";
-    private static final String STRING_LENGTH = "string.length";
+    public static final String ROW_NUM = "row.num";
```

Comment:
The `ROW_NUM` is each parallel num or total num?

Reply:
> Each parallel num

+1 @EricJoy2048